pycernan/avro/config.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import pycernan.config
+
+
+def host():
+    return os.getenv("PYCERNAN_AVRO_HOST", None) or pycernan.config.host()
```

Comment:
the `None` arg in `getenv` is intended for defaults, this can be written better as `os.getenv("PYCERNAN_AVRO_HOST",  pycernan.config.host())`.

Reply:
Updated.